lib/common.c

```diff
@@ -208,7 +245,7 @@ static int setup_struct_field_descriptor_set_btf(psabpf_btf_t *btf_md, psabpf_st
             }
         }
 
-        const struct btf_type *member_type = psabtf_get_type_by_id(btf_md->btf, md.effective_type_id);
+        member_type = psabtf_get_type_by_id(btf_md->btf, md.effective_type_id);
```

Comment:
Second type resolution? (see line 219, I think that both are the same if using `md.effective_type_id`)

Reply:
Done